chord_metadata_service/mcode/models.py

```diff
@@ -0,0 +1,276 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField, ArrayField
+from chord_metadata_service.restapi.models import IndexableMixin
+from chord_metadata_service.phenopackets.models import Gene
+from chord_metadata_service.patients.models import Individual
+from django.core.exceptions import ValidationError
+from chord_metadata_service.restapi.description_utils import rec_help
+import chord_metadata_service.mcode.descriptions as d
+from chord_metadata_service.restapi.validators import JsonSchemaValidator
+from chord_metadata_service.restapi.schemas import (
+    ONTOLOGY_CLASS, QUANTITY, COMPLEX_ONTOLOGY, TIME_OR_PERIOD, TUMOR_MARKER_TEST
+)
+
+
+############################ Field validators #############################
+
+ontology_class_validator = JsonSchemaValidator(ONTOLOGY_CLASS)
+quantity_validator = JsonSchemaValidator(schema=QUANTITY, format_checker=['uri'])
+tumor_marker_test_validator = JsonSchemaValidator(schema=TUMOR_MARKER_TEST)
+complex_ontology_validator = JsonSchemaValidator(schema=COMPLEX_ONTOLOGY, format_checker=['uri'])
+time_or_period_validator = JsonSchemaValidator(schema=TIME_OR_PERIOD, format_checker=['date-time'])
```

Comment:
I've already defined some of these in `validators.py` in the experiments PR that you just merged.